src/components/Welcome/index.tsx

```diff
@@ -10,31 +10,37 @@ import PageTitle from '../PageTitle'
 import ScreenLayout from '../Theme/ScreenLayout'
 import { DefaultContent, DocVideoContent } from './Content'
 import style from './style.scss'
-
+import balanceSrc from './assets/balance.svg'
 import type { StepComponentBaseProps } from '~types/routers'
 import type { StepTypes } from '~types/steps'
 
 const CAPTURE_STEP_TYPES: Set<StepTypes> = new Set([
   'document',
   'poa',
   'face',
+  'data',
   'auth',
 ])
 
 type WelcomeActionsProps = {
   customNextButtonLabel?: string
   nextStep: () => void
+  useWorkflow?: boolean
 }
 
 const WelcomeActions: FunctionComponent<WelcomeActionsProps> = ({
   customNextButtonLabel,
   nextStep,
+  useWorkflow,
 }) => {
   const { translate } = useLocales()
-
   const buttonLabel = customNextButtonLabel
     ? customNextButtonLabel
-    : translate('welcome.next_button')
+    : translate(
+        `${
```

Comment:
Any reason we are wrapping it in a string?

Reply:
seems to be complaining about not being string if I don't wrap it, as translate expects string as input. I think